src/SortableContainer/index.js

```diff
@@ -339,12 +346,13 @@ export default function SortableContainer(WrappedComponent, config = {withRef: f
 				if (transitionDuration) {
 					node.style[`${vendorPrefix}TransitionDuration`] = `${transitionDuration}ms`;
 				}
+
 				if (index > this.index && (sortingOffset + offset >= edgeOffset)) {
-					translate = -this.dimension;
+					translate = -this.dimensionWithMargins;
```

Comment:
Sorry for the delay, I think I missed the notification message for some reason :confounded:.
First-off margin collapsing only happens vertically (`margin-top` and `margin-bottom`), so we should add the horizontal margins (`margin-left` and `margin-right`).
```js
this.marginOffset = {
    x: this.margin.left + this.margin.right,
    y: Math.max(this.margin.top, this.margin.bottom)
}
```

As for using the max of the two margins, it only works if we assume all items have the same margins.
I had done a similar logic but after seeing all the possible cases I decided it's easier and cleaner to say we don't support margin collapsing.
That said I'm open to supporting this case (all item have the same margins).